Proofs/AES-GCM/GCMInitV8Sym.lean

```diff
@@ -0,0 +1,87 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Yan Peng
+-/
+import Proofs.«AES-GCM».GCMInitV8Pre
+import Tactics.Sym
+import Tactics.Aggregate
+import Specs.GCMV8
+
+namespace GCMInitV8Program
+
+abbrev H_addr (s : ArmState) : BitVec 64 := r (StateField.GPR 1#5) s
+abbrev Htable_addr (s : ArmState) : BitVec 64 := r (StateField.GPR 0#5) s
+
+set_option debug.byAsSorry true in
+set_option maxRecDepth 1000000 in
+set_option profiler true in
+theorem gcm_init_v8_program_run_152 (s0 sf : ArmState)
+    (h_s0_program : s0.program = gcm_init_v8_program)
+    (h_s0_err : read_err s0 = .None)
+    (h_s0_pc : read_pc s0 = gcm_init_v8_program.min)
+    (h_s0_sp_aligned : CheckSPAlignment s0)
+    (h_run : sf = run gcm_init_v8_program.length s0)
+    (h_mem : Memory.Region.pairwiseSeparate
+      [ ⟨(H_addr s0), 128⟩,
+        ⟨(Htable_addr s0), 2048⟩ ])
+    : read_err sf = .None := by
+  simp (config := {ground := true}) only at h_s0_pc
+  -- ^^ Still needed, because `gcm_init_v8_program.min` is somehow
+  --    unable to be reflected
+  sym_n 152
+  done
+
+-- TODO: this function is from Tests.Common
+-- need to put it in a better place
+def revflat (x : List (BitVec n)) : BitVec (n * x.length) :=
```

Comment:
Where do you guys suggest putting this function in?

Reply:
`Arm/BitVec.lean` is the "kitchen sink" file for such bitvec utils.